.github/workflows/ci.generate.ts

```diff
@@ -95,51 +95,41 @@ sudo mount --rbind /home /sysroot/home
 sudo mount -t proc /proc /sysroot/proc
 cd
 
-if [[ \`uname -m\` == "aarch64" ]]; then
```

Comment:
This is no longer necessary -- maybe because we're fixing up symlinks in the sysroot now?